lib/v2/nuist/yjs.js

```diff
@@ -0,0 +1,60 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const path = ctx.path === '/yjs/' ? 'index/tzgg' : ctx.path.replace(/^\/yjs\//, '');
+
+    const rootUrl = 'https://yjs.nuist.edu.cn';
+    const currentUrl = `${rootUrl}/${path}.htm`;
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+
+    const $ = cheerio.load(response.data);
+
+    let items = $('.gridlinediv')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+
+            const a = item.find('a').last();
+
+            return {
+                title: a.text(),
+                link: new URL(a.attr('href'), currentUrl),
```

Comment:
```suggestion
                link: new URL(a.attr('href'), currentUrl).href,
```
`tryGet` accepts a `string` key only. Otherwise, you will see this error when using RSSHub with Redis
```bash
Error message: TypeError: key.startsWith is not a function
    at getCacheTtlKey (/workspaces/RSSHub/lib/middleware/cache/redis.js:22:13)
    at get (/workspaces/RSSHub/lib/middleware/cache/redis.js:31:33)
    at Object.tryGet (/workspaces/RSSHub/lib/middleware/cache/index.js:59:27)
    at /workspaces/RSSHub/lib/v2/nuist/yjs.js:35:23
```

Reply:
Thanks for the tip 😃